tests/signals_connect.cc

```diff
@@ -8,7 +8,7 @@ using namespace aspect;
 
 template <int dim>
 void post_constraints_creation (const SimulatorAccess<dim> &,
-                                ConstraintMatrix &)
+                                aspect::ConstraintMatrix &)
```

Comment:
Can you revert this change and the corresponding changes in the other two files? I'm removing all of these uses of the class in #3738.